pyrefly.toml

```diff
@@ -0,0 +1,4 @@
+search-path = ["python", "tests/wpt/tests"]
+project-includes = ["python/tidy/**/*.py"]
+project-excludes = ["**/venv/**", "**/.venv/**", "tests/wpt/tests/**", "**/*test*.py", "**/tests/**"]
+python-interpreter = "./.venv/bin/python"
```

Comment:
Is it possible to avoid specifying this? Is prefly able to detect this from the environment like `ruff can?